src/app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item.tsx

```diff
@@ -34,9 +35,8 @@ export const StacksFungibleTokenAssetItem = forwardRef(
         caption={caption}
         data-testid={`asset-${name}`}
         imageCanonicalUri={imageCanonicalUri}
-        name={name}
         ref={ref}
-        subBalance={subBalance}
+        subBalance={unanchoredAssetBalance}
```

Comment:
The `rest as any` two lines below is eating up this type error,

```
src/app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item.tsx:39:9 - error TS2322: Type 'StacksFungibleTokenAssetBalance | undefined' is not assignable to type 'Money | undefined'.
  Type 'StacksFungibleTokenAssetBalance' is missing the following properties from type 'Money': amount, symbol, decimals

39         subBalance={unanchoredAssetBalance}
           ~~~~~~~~~~

  src/app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item.layout.tsx:24:3
    24   subBalance?: Money;
         ~~~~~~~~~~
    The expected type comes from property 'subBalance' which is declared here on type 'IntrinsicAttributes & StacksFungibleTokenAssetItemLayoutProps & RefAttributes<unknown>'
```

Reply:
Good spot, thanks